src/components/GlobalChatbot.tsx

```
@@ -0,0 +1,461 @@
+                      }`}>
+                        {message.role === 'user' ? <User className="h-3 w-3" /> : <Bot className="h-3 w-3" />}
+                      </div>
+                      
+                      <div className={`p-3 rounded-2xl text-sm ${
+                        message.role === 'user'
+                          ? 'bg-blue-500 text-white rounded-br-md'
+                          : 'bg-gray-100 dark:bg-gray-700 text-gray-900 dark:text-gray-100 rounded-bl-md'
+                      }`}>
+                        <div 
+                          className="whitespace-pre-wrap break-words"
+                          dangerouslySetInnerHTML={{ 
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)